coordinates/w128h64.json.example

```diff
@@ -188,6 +188,7 @@
     }
   },
   "standings": {
+    "font_name": "4x6",
```

Comment:
Just realized this will override the master font, so will need to update these where added.